AK/Windows.h

```diff
@@ -18,4 +20,40 @@
 #    undef IN
 #    pragma comment(lib, "ws2_32.lib")
 #    include <io.h>
+#    include <stdlib.h>
+
+inline void initiate_wsa()
+{
+    WSADATA wsa;
+    WORD version = MAKEWORD(2, 2);
+    int rc = WSAStartup(version, &wsa);
+    VERIFY(rc == 0 && wsa.wVersion == version);
+}
+
+inline void terminate_wsa()
+{
+    int rc = WSACleanup();
+    VERIFY(rc == 0);
+}
+
+static void invalid_parameter_handler(wchar_t const*, wchar_t const*, wchar_t const*, unsigned int, uintptr_t)
```

Comment:
static function in a header is no bueno -- even if it's empty.

It makes sense for these methods to be explicitly included into their respective main()s, but having them inline in each TU that has AK/Windows.h feels meh.

Non-blocking comment for sure though.